src/main/java/org/jabref/model/entry/BibEntryType.java

```diff
@@ -146,13 +149,14 @@ public int hashCode() {
         return Objects.hash(type, requiredFields, fields);
     }
 
+    /**
+     * This representation is also used in the UI, thus it is formatted more pretty
+     */
     @Override
     public String toString() {
-        return "BibEntryType{" +
-               "type=" + type +
-               ", requiredFields=" + requiredFields +
-               ", fields=" + fields +
-               '}';
+        return "type           = " + type + "," + OS.NEWLINE +
```

Comment:
I guess this will break many tests! Rather add a new method

Reply:
Introduced org.jabref.gui.importer.BibEntryTypePrefsAndFileViewModel to really fix that comment.

![image](https://github.com/JabRef/jabref/assets/1366654/e92160cd-bc3a-47f9-8468-e39cc673eda3)